### ISSUE TITLE
HSTS Webpage Error in HTTPS Websites

### ISSUE THREAD

**Issue by hasanalizxc** [wifiphisher/wifiphisher#1024]
Dear Creator,

I created network manager AP (fake wifi).

Just tested with HTTPS website and browser doesn't let me access the page and at the same time for redirection. There is no option "Go Forward" (maybe you know, sometimes if destination has unvalid certificate there is an option in website and you can choose "go forward" or "I trust this site").

Web Browser error like this:

Since the unix.stackexchange.com website uses HSTS, you cannot visit the site at this time. This page will probably work later because network errors and attacks are usually temporary.

And screenshot:

![83l43](https://user-images.githubusercontent.com/22418448/47324548-f75fe580-d668-11e8-9c65-de6a55722d08.png)

As you can see there is no option "Go Forward" or "I trust this site".

Could you please help us about this issue?

EDIT:

I found a website and i saw these lines but i couldn't understand how can we bypass this error in wifiphisher?

https://media.readthedocs.org/pdf/wifiphisher/latest/wifiphisher.pdf
![screenshot_20](https://user-images.githubusercontent.com/22418448/47324702-a0a6db80-d669-11e8-9bc4-0e1576cce8be.png)

Thanks in advance.

**Comment by sophron** [comment]
Hi @hasanalizxc,

What exactly you don't understand from the answer?

You cannot bypass this error. You can use other tools and trick the victim users into a domain that you own.

**Comment by sophron** [comment]
As mentioned on the answer, have a look at Evilngnx: https://github.com/kgretzky/evilginx2

**Comment by hasanalizxc** [comment]
Could you please make a PoC with Evilginx2? If you make this, people can understand more better and they can use these tools more effective.

**Comment by sophron** [comment]
No, I cannot. My time does not allow me to. Also keep in mind that Wifiphisher does not aim to be script kiddie friendly. An experienced tester can easily perform the previously mentioned attack.

**Comment by hasanalizxc** [comment]
:) You are so funny, i liked you. But use more polite words to people. Maybe your knowledge is better than me or anyone else but this is not means you need to kidding with people who knows less than you. Anyway, i know you are good person but you should select nice words for create a sentence.

**Comment by sophron** [comment]
I didn't mean to be aggressive or impolite.

**Comment by hasanalizxc** [comment]
Maybe i am newbie on security. Maybe i want to learn these scenarios. As i inderstand your talking you are better than me and i asked you politely made a video for two (2) scenarios but you told me this is not script kiddie's job. I just trusted you. That's all. I know your tool but i couldn't understand how can i work with evilhinx2 with your tool at the same time. What is the process order? When i will run evilginx2? ,etc. lots of questions. Anyway, thank you so much. I took your valuable time.
